tests/publiccloud/img_proof.pm

```diff
@@ -112,14 +112,15 @@ sub run {
     } else {
         $provider = $self->provider_factory();
         $instance = $provider->create_instance();
+
+        $instance->wait_for_guestregister();
```

Comment:
by this change you fully remove `wait_for_guestregister()` for all QAM runs , don't think that this was idea , right ?
also check will be only introduced for product runs

Reply:
Please re-check